docs/add_guest_credential.md

```diff
@@ -0,0 +1,37 @@
+# add_guest_credential
+
+Add a new guest credential to the Rubrik cluster.
+```py
+def add_guest_credential(username, password, domain=None, timeout=15):
+```
+
+## Arguments
+| Name        | Type | Description                                                                 | Choices |
+|-------------|------|-----------------------------------------------------------------------------|---------|
+| username | str  | The proxy username used for authentication.          |         |         |
```

Comment:
Do we need `proxy` in the `username` and `password` descriptions? To me it says we need some sort of proxy setup first and not just a service account.